script/deploy/BootstrapTestnet.s.sol

```diff
@@ -63,6 +64,7 @@ contract BootstrapTestnet is Script {
         proposals.push(Proposal(address(new zip012()))); /// MaxSupply settings proposal
         proposals.push(Proposal(address(new zip013()))); /// MaxSupply settings proposal
         proposals.push(Proposal(address(new zip014()))); /// MaxSupply settings proposal
+        proposals.push(Proposal(address(new zip015()))); /// MaxSupply settings proposal
```

Comment:
I think this should come after `zip018` as `zip015` is being simulated after `zip018`